src/foundry/client-esm/applications/apps/compendium-art-config.d.mts

```diff
@@ -18,7 +18,7 @@ declare class CompendiumArtConfig<
   Configuration extends ApplicationV2.Configuration = ApplicationV2.Configuration,
   RenderOptions extends ApplicationV2.RenderOptions = ApplicationV2.RenderOptions,
   RenderContext extends CompendiumArtConfigRenderContext = CompendiumArtConfigRenderContext,
-> extends HandlebarsApplicationMixin(ApplicationV2)<Configuration, RenderOptions, RenderContext> {
+> extends HandlebarsApplicationMixin(ApplicationV2)<RenderContext, Configuration, RenderOptions> {
```

Comment:
Good catch but could you also move around all things that extend HandlebarsApplicationMixin to also have `RenderContext` first?

Also is `RenderOptions` going to be more common to configure or `Configuration`?

Reply:
I expect Configuration to be more commonly adjusted than RenderOptions; that feeds directly into `this.options` while the RenderOptions are really just for calling `app.render(options)`, and I don't think people are really looking for special ways to specifically modify individual instances of an application in that way.